### PR TITLE
OSDOCS-11206: Update the 4.17 About this release RN Doc section

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.17

Issue:
[OSDOCS-11206](https://issues.redhat.com/browse/OSDOCS-11206)

Link to docs preview:
* [About this release](https://81256--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-about-this-release_release-notes)

Approvals will be similar to those on the equivalent [4.16 PR](https://github.com/openshift/openshift-docs/pull/77872).

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -29,10 +29,10 @@ You must use {op-system} machines for the control plane, and you can use either
 //Beyond this, Red{nbsp}Hat also offers a 12-month additional EUS add-on, denoted as _Additional EUS Term 2_, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is available on all architecture variants of {product-title}.
 
 //Odd-numbered release lifecycle verbiage
-The support lifecycle for odd-numbered releases, such as {product-title} 4.15, on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures is 18 months.
+The support lifecycle for odd-numbered releases, such as {product-title} {product-version}, on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures is 18 months.
```
Comment (julienlim) [inline release_notes/ocp-4-17-release-notes.adoc:32]: This statement only applies to odd numbered versions (and in the next 4.18 release, you'd have to revert this back to 4.17).

Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc:32]: Hi Ju. This paragraph will be commented out in the next release (even numbered). Another paragraph will be commented in.

Comment (julienlim) [reply release_notes/ocp-4-17-release-notes.adoc:32]: Got it

**Comment by julienlim** [comment]
lgtm

**Pull request merged**